exporter/zipkinexporter/config.go

```diff
@@ -27,5 +29,8 @@ var _ component.Config = (*Config)(nil)
 
 // Validate checks if the exporter configuration is valid
 func (cfg *Config) Validate() error {
+	if cfg.HTTPClientSettings.Endpoint == "" {
+		return errors.New("endpoint required")
+	}
```

Comment:
I think we should have a PR in core that adds a "Validate" on the `HTTPClientSettings` for this instead of duplicating everywhere.

Reply:
This is fo you to add an issue in core, and maybe work on it. Not to stop this PR .

Reply:
https://github.com/open-telemetry/opentelemetry-collector/issues/9132 is filed for this, thanks.